modules/afamqp/afamqp.c

```diff
@@ -348,6 +348,11 @@ afamqp_is_ok(AMQPDestDriver *self, const gchar *context, amqp_rpc_reply_t ret)
                     evt_tag_int("time_reopen", self->super.time_reopen));
           return FALSE;
         }
+    default:
```

Comment:
Can you add the numeric value of the unknown reply type, please?
And replace the "unknown server error" message to something unique, please?
E.g. "unknown response type" or "unknown response code"

Reply:
Done the mentioned changes..,

Reply:
Thanks! Looks good to me.